Makefile

```diff
@@ -28,8 +28,7 @@ ifeq ($(BUILD_IMAGE_NAME),)
 BUILD_IMAGE_NAME=otelopscol-build
 endif
 
-OTELCOL_BINARY=google-cloudops-opentelemetry-collector
-
+OTELCOL_BINARY=google-cloudops-opentelemetry-collector_$(GOOS)_$(GOARCH)$(EXTENSION)
```

Comment:
I'm not super familiar with make or bash, but for the case where:

- You are on Linux (GOOS defaults to linux)
- You run build-goo (which overrides GOOS to windows)

Does this build a binary called `google-cloudops-opentelemetry-collector_windows_...` or `google-cloudops-opentelemetry-collector_linux_...`?

(I'm not sure if the variable is set once initially or if this is re-run for each target that's called)

Can you check? If this works, then it's okay to leave as you have it

Reply:
addressed here: https://github.com/GoogleCloudPlatform/opentelemetry-operations-collector/pull/9#discussion_r464748585